### PR TITLE
UAVCAN ESC page restructure/generalise

### PR DESCRIPTION
**Pull request by hamishwillee**
Follow on from https://github.com/PX4/px4_user_guide/pull/547#issuecomment-518444859 - restructure for exensibility - common setup now common, with specific setup pull into own sections.

### CODE REVIEW

File: en/peripherals/uavcan_escs.md [hunk 1]
```diff
@@ -28,52 +31,53 @@ Sapog-based ESCs:
...
-> **Note** You may need to perform some tuning of motor control characteristics.
-  That is "out of the scope" and may be done later.
+## ESC Setup
 
-### Configuring PX4
+While UAVCAN devices are generally *plug'n'play* you will still need to enumerate (number) each of the ESC used in your system and set their direction so that they can be identified/controlled by PX4.
 
-Power the vehicle using the battery (you must power the whole vehicle, not just the flight controller!) and connect *QGroundControl*.
-Then navigate to the [Vehicle Setup > Parameters](../advanced_config/parameters.md) screen.
+> **Note** The ESC index and direction must match/map to the [Airframe Reference](../airframes/airframe_reference.md) for the vehicle type.
```
Comment (hamishwillee) [inline en/peripherals/uavcan_escs.md]: @bkueng @pavel-kirienko This is my GUESS on how PX4 understands/maps mixer definitions of motor order to ESC. Is it correct?

**Comment by pavel-kirienko** [comment]
This seems sensible. Is it possible to look at the text in its rendered form somewhere?

**Comment by hamishwillee** [comment]
@pavel-kirienko Sure - [rendered here](https://hamishwillee.gitbooks.io/ham_px4userguide/content/v/ed68cbd3d5cc75d522ab46c146b6ec9465c0d91e/en/peripherals/uavcan_escs.html)

**Comment by pavel-kirienko** [comment]
Thanks. I think it should be explicitly stated that the list of ESC is not exhaustive. For example, VESC and Myxa are not listed; in the future new ESC may appear and we don't want to lead readers to believe that there is nothing out there that's not listed here.

File: en/peripherals/uavcan_escs.md (hunk 1 above)
Comment (hamishwillee) [reply en/peripherals/uavcan_escs.md]: @pavel-kirienko Any thoughts on this?

**Comment by hamishwillee** [comment]
> Thanks. I think it should be explicitly stated that the list of ESC is not exhaustive. For example, VESC and Myxa are not listed; in the future new ESC may appear and we don't want to lead readers to believe that there is nothing out there that's not listed here.

Sure! Hoping to get back to this tomorrow. Can't merge without open questions closed https://github.com/PX4/px4_user_guide/pull/552#discussion_r311358794 because this is what allows you to use the ESC with non-Sapog hardware (otherwise there is no way you can know the right way to enumerate **for PX4**).

File: en/peripherals/uavcan_escs.md (hunk 1 above)
Comment (pavel-kirienko) [reply en/peripherals/uavcan_escs.md]: Your guess seems correct.

Comment (hamishwillee) [reply en/peripherals/uavcan_escs.md]: Thanks @pavel-kirienko. @cmic0 FYI if you know different let me know!

**Comment by hamishwillee** [comment]
@pavel-kirienko I think this addresses your last comment - ie expands out the list of products a bit and also adds a note that the list is not necessarily exhaustive.

Look OK to you?

File: en/peripherals/uavcan_escs.md [hunk 2]
```diff
@@ -107,6 +129,19 @@ This assigns the following Sapog configuration parameters for each enumerated ES
 
 > **Note** See [Sapog reference manual](https://files.zubax.com/products/io.px4.sapog/Sapog_v2_Reference_Manual.pdf) for more information about the parameters.
 
+### Myxa ESC Setup
+
+Motor enumeration for Myxa [Telega-based ESCs](https://zubax.com/products/telega) is usually performed using the [Kucher tool](https://files.zubax.com/products/com.zubax.kucher/) (or less "GUI-friendly" [UAVCAN GUI Tool](https://uavcan.org/GUI_Tool/Overview/)).
+
+There is some guidance here: [Quick start guide for Myxa v0.1](https://forum.zubax.com/t/quick-start-guide-for-myxa-v0-1/911) (Zubax blog)
+
+ (e.g Mitochondrik and Myxa) you can use the .
```
Comment (pavel-kirienko) [inline en/peripherals/uavcan_escs.md]: The what? :)

Comment (hamishwillee) [reply en/peripherals/uavcan_escs.md]: There goes my aura of perfection. Thanks!

**Pull request merged**